### DISCUSSION TITLE
Multiselect with pill boxes

### DISCUSSION THREAD

**Discussion by steve-mb** [livewire/flux#808] [Feature Requests]
I often make extensive use of the select2 multi select in my apps, it's so useful!  It would be great if Flux could do something similar, pill boxes when selecting multiple items.  
It's a great way to feedback to the user when using multiple selections.

https://select2.org/getting-started/basic-usage#multi-select-boxes-pillbox

**Comment by joshhanley** [comment]
@steve-mb thanks for the request! This is a duplicate of #513, so I'm going to close this one. Please add your support to that discussion.
